src/vstest.console/CommandLine/CommandLineOptions.cs

```diff
@@ -282,13 +283,29 @@ internal Framework TargetFrameworkVersion
         /// <param name="source">Path to source file to look for tests in.</param>
         public void AddSource(string source)
         {
+            var extnList = new List<string>() { ".xap", ".appx", ".dll", ".exe" };
```

Comment:
>extnList [](start = 16, length = 8)

make it const, & move it to top

Reply:
used static and readonly instead const because i was seeing compile errors